server/server.go

```diff
@@ -213,6 +214,10 @@ func (s *Server) Start() error {
 	s.tcpServer = s.runListener("tcp")
 	s.wgStart.Wait()
 	log.Info("Listeners started")
+
+	err := tlsoverridefirefoxsync.OverrideSyncStart(s.cfg.CanonicalSuffix)
+	log.Fatale(err, "Couldn't start Firefox override sync")
+
```

Comment:
change to return err

Reply:
Does the `service` package automatically exit when `Start()` returns a non-nil error?

Reply:
Empirically the answer is yes.  I'll make the change.

Reply:
Fixed.